### ISSUE TITLE
Replication

### ISSUE THREAD

**Issue by adsqueiroz** [philthynz/zabbix-mssql-2008-2012-powershell#4]
Hi man.

First I would like to thank you for your template, it is very good and is helping a lot.
I am trying to use then also to get Replications, but I am see that the script that get the MSSQL instances do not get the informations. I am researching to understand how is possible get this king of information, but I did not get success yet.

My question is. Do you now how to get informations of Replication using PowerShell? Or where is possible find on Windows Register?

If I have some success I will let you now.

Thx

**Comment by philthynz** [comment]
Hey @adsqueiroz 

No sorry I haven't used Zabbix to get DB replication metrics. However it might be possible via Process counters. The powershell in this template mostly uses process counters.

I would start by listing all the available process counters on the server and see if there is one for replication. If there is you could use them in the same way this template does.

Cheers! Glad you like the template.

**Comment by adsqueiroz** [comment]
Hey @philthynz 

no worry man.

I got the full list of the process counters, using: **typeperf -qx > C:\zabbix\items.txt**  and I saw that bring replication, for example:

\MSSQL$DISTRELUZ:Replication Dist.(MY-TEST-REPLICATION)\Dist:Delivery Latency

I tried reproduce on the template, creating a item:

perf_counter["\{#PERFCOUNTERNAME}:Replication Dist.({#INST})\Dist:Delivery Latency"]

but the variable #INST gets the value of the database instance, not the replications instance (I am trying to understand how to get and create a new variable on the script). And analysing the script called "mssql_instances.ps1" I could understand the reason.

And creating items using the process counters that exists in my server, for example:

\MSSQL$DISTRELUZ:Replication Agents(Queuereader)\Running
\MSSQL$DISTRELUZ:Replication Agents(Merge)\Running
\MSSQL$DISTRELUZ:Replication Agents(Distribution)\Running
\MSSQL$DISTRELUZ:Replication Agents(Logreader)\Running
\MSSQL$DISTRELUZ:Replication Agents(Snapshot)\Running

I had success, those items bring values from the system. So your template is working very well.

I think your tip and your reply. 

Cheers! Your template is GREAT!
